pf/proto/object.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016-2024, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proto
+
+import (
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+
+	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
+)
+
+var (
+	_ = shim.Resource(object{})
+	_ = shim.SchemaMap(attrMap{})
+)
+
+type object struct {
+	pseudoResource
+	obj tfprotov6.SchemaObject
```

Comment:
SchemaObjectNestingMode handled? Looks pretty suspect.

Reply:
It is not handled in [506589f](https://github.com/pulumi/pulumi-terraform-bridge/pull/1973/commits/506589f50ae024783592c84b70987a2915c27071).

Reply:
This should be fixed in [deed5e7](https://github.com/pulumi/pulumi-terraform-bridge/pull/1973/commits/deed5e7ce50778d2a540ff5ef833f357f38c915a).